src/app/authentication/repo/customerRepo.go

```diff
@@ -0,0 +1,45 @@
+package repo
+
+import (
+	"authentication/constants"
+	"authentication/utils/db"
+
+	"log"
+)
+
+func IsEmailExist(email string) bool {
+
+	DatabaseConnection, err := db.ConnectionWithDb()
+
+	if err != nil {
+		log.Fatalf("%s :%s", constants.ErrConnectingDB.Error(), err)
+	}
+
+	query := "SELECT COUNT(*) FROM customerdata WHERE email = ?"
```

Comment:
```suggestion
	if err != nil {
		log.Fatalf("%s :%s", constants.ErrConnectingDB.Error(), err)
	}
	query := "SELECT COUNT(*) FROM customerdata WHERE email = ?"
```

Remove unnecessary spaces